data/json/regional_map_settings.json

```diff
@@ -96,6 +97,7 @@
           "t_tree_birch": 16,
           "t_tree_elm": 16,
           "t_tree_cottonwood": 16,
+          "t_tree_basswood": 16,
```

Comment:
Looks to me like you forgot to make f_wintergreen spawn.

Reply:
No, this is still a work in progress. I need to figure out if I make it accessible via foraging only (you randomly get leaves and berries in autumn and winter) or rarely spawn it in the dense forest. Probably both.